scripts/src/utils/utils.ts

```diff
@@ -3,7 +3,10 @@ import { join } from "path";
 import * as fs from "fs";
 
 import { Express } from "express";
-import { Context } from "./public/routes";
+import { Context } from "../public/routes";
+
+import { path } from "./path";
```

Comment:
Do you need it here?

Reply:
It seems like fix of previous mistake. Context type is used in row 11

Reply:
you're only using `path` for `fromProjectRoot` which you should remove too